### PR TITLE
Added product-default to the pem search (#1624)

### LINKED ISSUE #1624: Look for product certs in both the /etc/pki/product-default and /etc/pki/product
### Is your feature request related to a problem?
Previously, RHEL placed product certs in the /etc/pki/product directory.  Recently, /etc/pki/product-default was added as a potential location.

### Describe the solution you'd like
Look for product certs in both the /etc/pki/product-default and /etc/pki/product

___

## Acceptance Criteria:

- [ ] Certs in both the /etc/pki/product-default and /etc/pki/product are detected

### PR DESCRIPTION
**Pull request by myersCody**
Closes #1624

### CODE REVIEW

**Comment by kholdaway** [comment]
Nice job!

File: roles/jboss_ws/tasks/main.yml [hunk 1]
```diff
@@ -74,34 +74,48 @@
...
+
+- name: check for jws cert in product-default directory
+  raw: ls /etc/pki/product-default/185.pem 2>/dev/null
+  register: jws_product_default
+  ignore_errors: yes
+  when: 'jboss_ws'
+
+- name: combine jws certs
+  set_fact:
+    jws_has_certs: "{{
```
Comment (kholdaway) [inline roles/jboss_ws/tasks/main.yml]: An extra s was added

Comment (abaiken) [inline roles/jboss_ws/tasks/main.yml]: Now that this is named jws_has_certs instead of jws_has_cert we might want to update the processing.

Comment (myersCody) [reply roles/jboss_ws/tasks/main.yml]: Opps I think we should just rename it back to jws_has_cert. Because it will still only find one cert.

Comment (abaiken) [reply roles/jboss_ws/tasks/main.yml]: That's totally fine but you're going to want to do a lot of running actual scans to make sure that it works as expected (scan a system that has a pem file in each directory/ a pem file in either directory/ etc.).

Comment (myersCody) [reply roles/jboss_ws/tasks/main.yml]: Done. Kev informed me that there were multiple ways to detect jboss_ws. So I have ssh into a machine without jboss present and added /etc/pki/product-default/185.pem. Just tested and I got present for jboss_ws

Comment (kholdaway) [reply roles/jboss_ws/tasks/main.yml]: This definitely is better.  You will notice that we missed a lot of 69.pem before this PR.  Wow! Here is a comparison of 0.0.44 to this PR (edited a bit):
```
Systems in 44-large-network.json but not 1624-large-network.json: 

#### dhcp-10-12-211-111.hq.gsslab.rdu.redhat.com--00:50:56:9E:4F:3D - (system1, system2) ####
mismatch: redhat_certs (69.pem,185.pem;69.pem;)

#### dhcp-10-12-211-130.hq.gsslab.rdu.redhat.com--00:50:56:9E:A1:E9 - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp-10-12-211-33.hq.gsslab.rdu.redhat.com--00:50:56:9E:7A:2C - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp-10-12-211-93.hq.gsslab.rdu.redhat.com--00:50:56:9E:97:EB - (system1, system2) ####
mismatch: redhat_certs (201.pem,201.pem;69.pem;)

#### dhcp181-135.gsslab.rdu2.redhat.com--00:50:56:9E:02:E6 - (system1, system2) ####
mismatch: redhat_certs (183.pem,183.pem;69.pem;)

#### dhcp181-62.gsslab.rdu2.redhat.com--00:50:56:9E:CF:A6 - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp182-157.gsslab.rdu2.redhat.com--00:50:56:9E:C8:C6 - (system1, system2) ####
mismatch: redhat_certs (183.pem,183.pem;69.pem;)

#### dhcp182-33.gsslab.rdu2.redhat.com--00:50:56:9E:39:EB - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)

#### dhcp182-37.gsslab.rdu2.redhat.com--00:50:56:9E:3A:1F - (system1, system2) ####
mismatch: redhat_certs (None,69.pem;)
```

**Comment by kholdaway** [comment]
I am happy with these changes.  I like the final PR much better.  Nice cleanup.

**Comment by kholdaway** [comment]
Let @abaiken take a look before merging.

File: roles/jboss_ws/tasks/main.yml (hunk 1 above)
Comment (myersCody) [reply roles/jboss_ws/tasks/main.yml]: 🥇 Nice catch by dan to find the product-default directory

**Pull request merged**